pkg/service/HelmAppService.go

```diff
@@ -534,6 +500,8 @@ func (impl HelmAppServiceImpl) InstallRelease(ctx context.Context, request *clie
 		UpgradeCRDs:      true,
 		CreateNamespace:  true,
 	}
+
+	impl.logger.Debug("Installing release")
```

Comment:
add some contextual info in log

Reply:
Contextual info is logged in the caller method in HelmAppServiceWrapper.go class. This log is just for to know which step currently code is executing (for debugging)

Reply:
having contextual info in log will help in debugging

Reply:
done